xe/do.py

```diff
@@ -33,6 +36,6 @@ def run(cmd):
 
     print('Running: %s' % cmd)
     try:
-        call(cmd)
+        call(cmd, shell=(os.name == 'nt'))
```

Comment:
I actually don't remember why I put this in there. Works on my machine with `shell=False` on Windows. I'll revert this line to `call(cmd)`.